### PR TITLE
validate BIRD

### PR DESCRIPTION
**Pull request by Solid-Proof**
# Validate [{BIRD}](https://solscan.io/token/7y2KpMzsxmjCN8ogA3PSfwZvmYUgMuWo7NWY4Ct5jRHp)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/BirdOnSolana/status/1711721359901118693
- Coingecko/ CMC URL (If available): Not available

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by Timoon21** [comment]
Hi, can you add your token data at the end of the file (at the last line)? Not at a random place. Thanks.

**Comment by Solid-Proof** [comment]
Screenshot 2023-10-31 180657
Hey , updated the token meta data listed as per requested. Please check

File: validated-tokens.csv [hunk 1]
```diff
@@ -790,4 +790,5 @@ solana green,green,7VUUicqKYFVmewg1U4w4e9K4FoEYdTcYyxYCPhRNf5Qy,9,https://arweav
 WEN,$WEN,3J5QaP1zJN9yXE7jr5XJa3Lq2TyGHSHu2wssK7N1Aw4p,6,https://arweave.net/sfrKTJisIlWsNJfw6GWnsitsbUbN6WIjoVJZWMe2Mdk,true
 Water Bears,WTRBR,Aymdf5Fp2URJNcsHpEF2NqEqFHiqvvhrDvYQtyZFw7zn,4,https://arweave.net/GZNQl83S1kK5kvak1yvMrI8muuPVm80fdliPcgwsWfw,true
 TEST,$TEST,D8TFo4FsDF9CnejkezEZtwdhdmwaiNgLRDAQUTwWQuBU,6,https://bafkreia7lr4higfpv6xc5owc3m2vjgcq2zzb7nqoaeut2lqcfy4pplpy5u.ipfs.nftstorage.link,true
-CHAD,CHAD,G7rwEgk8KgQ4RUTnMy2W2i7dRDq4hXHD4CSp9PSmSbRW,9,https://bafybeicl7hj53czkh3vqclya2tltuuj3f4r7acwbchwjjajfv3jvff62iu.ipfs.nftstorage.link/,true
\ No newline at end of file
+CHAD,CHAD,G7rwEgk8KgQ4RUTnMy2W2i7dRDq4hXHD4CSp9PSmSbRW,9,https://bafybeicl7hj53czkh3vqclya2tltuuj3f4r7acwbchwjjajfv3jvff62iu.ipfs.nftstorage.link/,true
+SOLBIRD,BIRD,7y2KpMzsxmjCN8ogA3PSfwZvmYUgMuWo7NWY4Ct5jRHp,6,https://fpzbucl4bp2yiez4thmx56fx2nutnfqqbuhptm4sfs5hwrr2myna.arweave.net/K_IaCXwL9YQTPJnZfvi302k2lhANDvmzkiy6e0Y6Zho,true
```
Comment (Timoon21) [inline validated-tokens.csv:794]:
```suggestion
SOLBIRD,SolBird,7y2KpMzsxmjCN8ogA3PSfwZvmYUgMuWo7NWY4Ct5jRHp,6,https://arweave.net/K_IaCXwL9YQTPJnZfvi302k2lhANDvmzkiy6e0Y6Zho,true
```

**Review by Timoon21 — APPROVED** [review]
- [x] On-chain metadata matches the PR
- [x] Token address matches the tweet
- [x] No duplication on Jupiter's strict list
- [x] Community validation

**Pull request merged**